.vortex/installer/tests/Fixtures/install/provision_database_lagoon/docs/deployment.md

```
@@ -0,0 +1,13 @@
+@@ -16,3 +16,12 @@
+    deployment.
+ 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)